<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_ManualMakeCard</class>
 <widget class="QWidget" name="Sys_ManualMakeCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: &quot;思源黑体 CN Regular&quot;;
font-size: 30px;
font-weight: bold;
line-height: 49px;
letter-spacing: 1px;
color: 0
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_29">
     <item>
      <widget class="QPushButton" name="pushButton_ReadID">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>读取身份证</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Debug">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox::indicator {width: 24px;height: 24px;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_WithoutIDCard">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox::indicator {width: 24px;height: 24px;}</string>
       </property>
       <property name="text">
        <string>无证办卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_NewCard">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
       </property>
       <property name="text">
        <string>新制卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ReplaceCard">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
       </property>
       <property name="text">
        <string>补办卡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_QueryCardInfo">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>查询卡信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_LoadCardData">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>加载数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_MakeCard">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>开始制卡</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QRadioButton" name="radioButton_commitInfo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
          </property>
          <property name="text">
           <string>提交数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_PremakeCard">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
          </property>
          <property name="text">
           <string>开户制卡</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_WriteCard">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
          </property>
          <property name="text">
           <string>写入数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_PrintCard">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
          </property>
          <property name="text">
           <string>打印卡面</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_EnableCard">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
          </property>
          <property name="text">
           <string>启用卡片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_Photo">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>360</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(124, 124, 124);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(170, 0, 0);</string>
               </property>
               <property name="text">
                <string>姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Name">
               <property name="minimumSize">
                <size>
                 <width>320</width>
                 <height>40</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="topMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(170, 0, 0);</string>
               </property>
               <property name="text">
                <string>性别</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QRadioButton" name="radioButton_Male">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
                 </property>
                 <property name="text">
                  <string>男</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::MinimumExpanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_Female">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QRadioButton::indicator { 
	width: 24px;
	height: 24px;
}
QRadioButton::indicator::unchecked {
	image: url(:/Image/RadioUnChecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/Image/RadioCheck.png);
}
</string>
                 </property>
                 <property name="text">
                  <string>女</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(170, 0, 0);</string>
               </property>
               <property name="text">
                <string>手机号码</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Mobile">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>40</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(170, 0, 0);</string>
               </property>
               <property name="text">
                <string>民族</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_Nationality">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>240</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>身份证号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_CardID">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>40</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>证件有效期</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_IssuedDate">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65535</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ExpiredDate">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65535</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>社保卡号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditl_SSCard">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>40</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Status">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>65535</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>发卡银行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Bank">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>40</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_Tips">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 0);</string>
       </property>
       <property name="text">
        <string>友情提醒:
    无卡办证时,红色字体部分为必选，不可为空!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1395</width>
       <height>79</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
